timely/src/execute.rs

```diff
@@ -269,8 +302,65 @@ pub fn execute_from_args<I, T, F>(iter: I, func: F) -> Result<WorkerGuards<T>,St
     where I: Iterator<Item=String>,
           T:Send+'static,
           F: Fn(&mut Worker<Allocator>)->T+Send+Sync+'static, {
-    let configuration = Configuration::from_args(iter)?;
-    execute(configuration, func)
+    let mut opts = getopts::Options::new();
```

Comment:
The opts parsing .. I was sort of imagining as part of `ExecuteConfig`. That way e.g. someone could parse up some arguments, and then maybe flip a few, read out some and print them on screen, stuff like that. Is there a reason to stash this logic here, vs in some constructor/initialization for `ExecuteConfig`?